app/org/views.py

```diff
@@ -19,6 +19,9 @@ def view_org(org_id):
     organization = Organization.query.filter_by(id=org_id).first()
     user = User.query.filter_by(id=organization.user_id).first()  
     pics = []
+    user = User.query.filter_by(id=organization.user_id).first()
+    if user.approved is False:
+        flash('The admin has not approved your organization', 'error') 
```

Comment:
edit flash content to: `the admin has not approved your profile. Your organization will not be visible publicly until you receive approval`